### PR TITLE
add sample bookinfo demo for servicemesh

### PR DESCRIPTION
**Pull request by zackzhangkai**
add sample bookinfo demo for servicemesh

Signed-off-by: zackzhangkai <zackzhang@yunify.com>

### CODE REVIEW

File: src/test/sample-bookinfo/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: v1
```
Comment (hlwanghl) [inline src/test/sample-bookinfo/Chart.yaml:18]: please align with the [semantic versioning](https://semver.org/), i.e. something like `version: 1.0.0`.

**Comment by zackzhangkai** [comment]
@zryfish @hlwanghl please review this, if there is no problem, please merge it. Thx.

**Pull request merged**